app/helpers/commentaire_helper.rb

```diff
@@ -7,9 +7,9 @@ def commentaire_is_from_me_class(commentaire, connected_user)
 
   def commentaire_answer_action(commentaire, connected_user)
     if commentaire.sent_by?(connected_user)
-      "Envoyer un message à l’instructeur"
+      I18n.t('helpers.commentaire_helper.send_message_instructor')
```

Comment:
Pour les helpers, il n'y a pas de magie ou de convention spécifique où le chemin de la clef est inféré automatiquement (comme dans les contrôleurs ou les vues).

Mais éventuellement je dirais qu'on pourrait faire pareil que pour les contrôleurs, et omettre le suffixe `_helper` dans le chemin de la clef.

```ruby
      I18n.t('helpers.commentaire.send_message_to_instructor')
```

Oh, et aussi on garde généralement les termes métier en français dans le code – et c'est le cas de "instructeur". On pourrait donc nommer la clef comme ça :

```suggestion
      I18n.t('helpers.commentaire.send_message_to_instructeur')
```